maps/models.py

```diff
@@ -525,6 +525,45 @@ class Map(models.Model):
         related_name="population_movements_source_for",
     )
 
+    # Severity, analyses and evolution group
+    # Severity, and composite analysis block
```

Comment:
Yes first comment shows that it's the start of a new group, second comment shows the block of fields inside that group.